dill/_dill.py

```diff
@@ -1037,12 +939,9 @@ def _create_filehandle(name, mode, position, closed, open, strictio, fmode, fdat
         import tempfile
         f = tempfile.TemporaryFile(mode)
     else:
-        # treat x mode as w mode
```

Comment:
I think this comment should not be removed

Reply:
Maybe put it next to the `and ("w" in mode or "x" in mode):` line below?